content/blog/puluminaries-program/index.md

```diff
@@ -0,0 +1,78 @@
+
+#### What does it mean to be a Pulumi Puluminary?
+
+Pulumi Puluminaries serve as champions of cloud engineering. They help users navigate the complexities of building, deploying, and managing applications with Pulumi. They also promote the benefits of Infrastructure as Code in modern organizations by creating thought leadership content, hosting or co-hosting events, and offering guidance for those just getting started.
+
+**Key Responsibilities for Pulumi Puluminaries:**
+
+As a Pulumi Puluminary, you will be expected to engage with the community atleast once a quarter. Here are some ways you can contribute:
+
+1. **Community Engagement**
+    - Actively share Pulumi-related articles, tutorials, and project stories.
+    - Present at conferences, meetups, or webinars on topics that highlight Pulumi’s value.
+    - Mentor new users and help them get started with Pulumi.
+    - Participate in Pulumi’s community forums, Slack channels, and social media discussions.
+
+2. **Thought Leadership**
+    - Write blog posts (like [Simen A. W. Olsen](/blog/author/simen-a-w-olsen/) did in his latest post [Your Perfect Infrastructure May Not Be So Perfect](/blog/your-perfect-infrastructure/) or [Tyler Mulligan](/author/tyler-mulligan/)) or create [videos showcasing](https://www.youtube.com/playlist?list=PLyy8Vx2ZoWlqxDJjRRhgLGu1_Oct0VVhN) innovative ways to use Pulumi.
+    - Contribute to discussions around best practices and patterns for cloud engineering with Pulumi.
+
+3. **Product Feedback**
+    - Offer beta-testing support and provide product feedback directly to the Pulumi team.
+    - Participate in early-access programs and help guide product roadmap decisions.
+
+#### Benefits of Joining the Program
+
+- **Early Access**
+  Get sneak peeks at new Pulumi features and releases before they are publicly announced.
+- **Direct Collaboration**
+  Collaborate one-on-one with Pulumi engineers and product managers, influencing future capabilities.
+- **Recognition and Swag**
+  Receive official Pulumi Puluminaries 2.0 digital badges, exclusive swag, and spotlight opportunities on Pulumi’s blog and at events.
+- **Networking and Growth**
+  Build relationships with peers in the Pulumi ecosystem, deepen your technical expertise, and grow your professional network.
+
+#### How to Get Involved
+
+We are looking for Pulumi enthusiasts who have shown dedication and passion for empowering others in the Pulumi community. Whether you have built cutting-edge projects, delivered insightful conference talks, or helped users overcome technical hurdles, we would love to hear your story.
+
+Stay tuned for more details on how to apply or nominate someone for the Pulumi Puluminaries 2.0 Program. In the meantime, keep innovating and sharing your Pulumi journey, and do not forget to learn more about our current Pulumi Puluminaries [here](/community/puluminaries/).
```

Comment:
we have a nomination form. just give them the form so they can apply. Also use a big CTA button if you know how to put one in